content/graphql/essentials/compatibility.md

```diff
@@ -0,0 +1,16 @@
+# Compatibility
```

Comment:
Rename essentials to **GraphQL basics**

Reply:
Voir si on conserve **set-up and usage** ou on le renomme

Dans essantials => il manque un lien pour continuer la découverte

Compatibility => sortir la recommendation dans un sous meny USage recommandation

Sur tt les sections rajouté un lien vers la section suivante

Limitations à déplacer dans la section **set-up and usage**

Reply:
Queries examples => **Use cases & usages** (on rajoute dedans working with pagination, customize output field aliasing)